src/dataflow/render/mod.rs

```diff
@@ -77,11 +79,8 @@ pub(crate) fn build_local_input<A: Allocate>(
                 Some(&index_id.to_string()),
             );
             match context.arrangement(&get_expr, &index.keys) {
-                Some(ArrangementFlavor::Local(local)) => {
-                    manager.set(
-                        index_id,
-                        WithDrop::new(local.trace, Rc::new(None::<source::SourceToken>)),
-                    );
+                Some(ArrangementFlavor::Local(oks, errs)) => {
+                    manager.set(index_id, TraceBundle::new(oks.trace, errs.trace));
```

Comment:
This may be minor, but it was intentional that it was previously noisy to create a `WithDrop` with nothing to drop (I think a few bugs were folks/me not collecting the right tokens). Maybe this has quieted down enough that it isn't worth making noisy again.

Reply:
Refreshing my understanding of `WithDrop`, I don't think this was actually the case before this PR (the `from` function was the "easy" way to make something without a thing to drop).